src/backend/core/admin.py

```diff
@@ -82,6 +82,15 @@ class UserAdmin(auth_admin.UserAdmin):
         ),
         (_("Important dates"), {"fields": ("created_at", "updated_at")}),
     )
+    add_fieldsets = (
+        (
+            None,
+            {
+                "classes": ("wide",),
```

Comment:
Isn't that a style attribute ? I'm curious : why add it here ?

Reply:
I overrode default values from the base class [here](https://github.com/django/django/blob/3cadeea077a98367a4ed344d645df0aff243de91/django/contrib/auth/admin.py#L64). I replaced the field username by email, and kept others values as it was. Wdyt?

Reply:
Smart ! OK, thanks then, curiosity satiated.